### ISSUE TITLE
[Tarea 3] Tamaño del input

### ISSUE THREAD

**Issue by jvlara** [IIC2213/Syllabus-2021-1-#30]
Hola, 
¿podemos asumir que el tamaño del input de vértices es fijo? ¿o nuestra maquina debe funcionar para cualquier tamaño de input arbitrario?,
Esto por la cantidad de estados, ya que si tengo una maquina fija puedo trabajar con una maquina especialista en ese tamaño.

**Comment by ricardor4** [comment]
No, no puedes, tu máquina debe funcionar para un input arbitrario.

Saludos
